application/server/server_test.go

```diff
@@ -1070,18 +1072,50 @@ func runSmokeTest(t *testing.T, repo string, commit string, file1 string, file2
 		if !ok || codeIssueData["hasAIFix"] == false {
 			continue
 		}
-		call, err := loc.Client.Call(ctx, "workspace/executeCommand", sglsp.ExecuteCommandParams{
+		call, e := loc.Client.Call(ctx, "workspace/executeCommand", sglsp.ExecuteCommandParams{
 			Command:   snyk.CodeFixDiffsCommand,
 			Arguments: []any{uri.PathToUri(scanParams.FolderPath), uri.PathToUri(issue.FilePath), issue.Id},
 		})
-		assert.NoError(t, err)
+		assert.NoError(t, e)
 		var unifiedDiffs []code.AutofixUnifiedDiffSuggestion
-		err = call.UnmarshalResult(&unifiedDiffs)
-		assert.NoError(t, err)
+		e = call.UnmarshalResult(&unifiedDiffs)
+		assert.NoError(t, e)
 		assert.Greater(t, len(unifiedDiffs), 0)
 		// don't check for all issues, just the first
 		break
 	}
+
+	checkFeatureFlagStatus(t, &loc)
+}
+
+func checkFeatureFlagStatus(t *testing.T, loc *server.Local) {
+	t.Helper()
+
+	call, err := loc.Client.Call(ctx, "workspace/executeCommand", sglsp.ExecuteCommandParams{
+		Command:   snyk.GetFeatureFlagStatus,
+		Arguments: []any{"bitbucketConnectApp"},
+	})
+
+	if err != nil {
+		t.Fatal("FeatureFlagStatus command failed", err)
+	}
+
+	if err := call.Error(); err != nil {
+		log.Warn().Err(err).Msg("Received an error response from FeatureFlagStatus command")
+	}
```

Comment:
question: why is this a `warn` and not an `error`

Reply:
I've been struggling with how to deal with a 403 response. A `403` is not an error, just lack of permissions. I didn't want to treat it as a cause of test failure, but rather as an expected possible outcome.
I'm refactoring it now, one moment 🙏